USBHelperLauncher/Program.cs

```diff
@@ -195,6 +196,40 @@ static void Main(string[] args)
                 }
             }
 
+            // Make sure that FiddlerCore's key container can be accessed
+            string keyContainerName = FiddlerApplication.Prefs.GetStringPref("fiddler.certmaker.bc.KeyContainerName", "FiddlerBCKey");
+            try
+            {
+                CspParameters cspParams = new CspParameters();
+                cspParams.KeyContainerName = keyContainerName;
+                var _ = new CspKeyContainerInfo(cspParams).UniqueKeyContainerName; // this will throw an exception if the container cannot be accessed
+            }
+            catch (CryptographicException)
+            {
+                using (MD5 md5 = MD5.Create())
+                {
+                    byte[] hash = md5.ComputeHash(Encoding.ASCII.GetBytes(keyContainerName.ToLower() + "\0"));
+                    var reader = new BinaryReader(new MemoryStream(hash));
+                    var sb = new StringBuilder();
+                    for (int i = 0; i < 4; i++)
+                    {
+                        sb.AppendFormat("{0:x8}", reader.ReadInt32());
+                    }
+                    string hashString = sb.ToString();
+
```

Comment:
Remove blank line. You could add it to the line above to space out the declarations if that's what you were going for.